templates/issue.html

```diff
@@ -292,41 +300,29 @@
 					<input name="issue_number" value="{{.IssueNumber}}" hidden></input>
```

Comment:
issue number is unnecessary. Just get it from params

Reply:
done